spring-data-relational/src/main/java/org/springframework/data/relational/core/sql/SqlIdentifier.java

```diff
@@ -59,13 +60,13 @@ public SqlIdentifier transform(UnaryOperator<String> transformationFunction) {
 		}
 
 		@Override
-		public String toSql(IdentifierProcessing processing) {
-			throw new UnsupportedOperationException("An empty SqlIdentifier can't be used in to create SQL snippets");
+		public String getReference() {
```

Comment:
Please do not change the order of methods.

It makes reading the diffs unnecessary hard.

Reply:
Yeah, sure, I have fixed this with [this](https://github.com/spring-projects/spring-data-jdbc/pull/1111/files#diff-ad9a19ceaf2639443aa86e26bc0cd97e5e9b2aad5ea158cae9477246b5d38294L79) commit